mapbox/dependencies.gradle

```diff
@@ -4,17 +4,17 @@ ext {
     MasVersionName = "2.1.0-SNAPSHOT"
 
     minSdkVersion = 15
-    targetSdkVersion = 25
+    targetSdkVersion = 23
     compileSdkVersion = 25
-    buildToolsVersion = "25.0.2"
+    buildToolsVersion = "23.0.3"
 
     dep = [
             // Support
-            supportAppcompat     : 'com.android.support:appcompat-v7:25.1.0',
-            supportCompat        : 'com.android.support:support-compat:25.1.0',
-            supportDesign        : 'com.android.support:design:25.1.0',
-            supportRecyclerview  : 'com.android.support:recyclerview-v7:25.1.0',
-            supportCardView      : 'com.android.support:cardview-v7:25.1.0',
+            supportAppcompat     : 'com.android.support:appcompat-v7:23.4.0',
+//            supportCompat        : 'com.android.support:support-compat:23.4.0',
```

Comment:
@LornTao you don't need to comment out this line, it's just a version definition.